### PR TITLE
Merge Beta to Stage

### PR DESCRIPTION
**Pull request by ravinderwalkover**
Merge Beta to Stage

### CODE REVIEW

File: apps/web-giddh/src/app/shared/header/header.component.ts [hunk 1]
```diff
@@ -808,19 +742,7 @@ export class HeaderComponent implements OnInit, AfterViewInit, OnDestroy, AfterV
...
-                // this.datePickerOptions = { ...this.datePickerOptions, startDate: moment(dateObj[0]), endDate: moment(dateObj[1]), chosenLabel: dateObj[2]};
-                // const from: any = moment().subtract(30, 'days').format(GIDDH_DATE_FORMAT);
-                // const to: any = moment().format(GIDDH_DATE_FORMAT);
-                // const fromFromStore = moment(dateObj[0]).format(GIDDH_DATE_FORMAT);
-                // const toFromStore = moment(dateObj[1]).format(GIDDH_DATE_FORMAT);
-
-                // if (from === fromFromStore && to === toFromStore) {
-                //     this.isTodaysDateSelected = true;
-                // }
+                this.isTodaysDateSelected = !dateObj[3];  //entry-setting API date response in case of today fromDate/toDate will be null
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/header/header.component.ts:745]: @rohit-walkover can we take your today story changes to PROD?

File: apps/web-giddh/src/app/store/authentication/authentication.reducer.ts [hunk 2]
```diff
@@ -580,7 +580,7 @@ export function SessionReducer(state: SessionState = sessionInitialState, action
                 }
                 let fromDate: any = data.fromDate ? moment(data.fromDate, GIDDH_DATE_FORMAT) : moment().subtract(30, 'days');
                 let toDate: any = data.toDate ? moment(data.toDate, GIDDH_DATE_FORMAT) : moment();
-                latestState.applicationDate = [fromDate._d, toDate._d, chosenLabel];
+                latestState.applicationDate = [fromDate._d, toDate._d, chosenLabel, !!data.fromDate ];
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/store/authentication/authentication.reducer.ts:583]: @rohit-walkover ?

File: apps/web-giddh/src/app/shared/aside-menu-other-taxes/aside-menu-other-taxes.html [hunk 3]
```diff
@@ -13,7 +13,7 @@ <h3 class="aside-title">Tax applicable</h3>
 
                 <div class="col-xs-12 col-sm-6">
 
-                    <div style="display: flex;flex-direction: column" class="selectTaxDropdown">
+                        <div class="selectTaxDropdown d-flex flex-column">
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/shared/aside-menu-other-taxes/aside-menu-other-taxes.html:16]: Avoid camelCase names

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/assets/css/typography.css [hunk 4]
```diff
@@ -61,10 +61,10 @@
...
 
-$opensans-regular: "opensans-regular";
-$opensans-semibold: "opensans-semibold";
-$opensans-bold: "opensans-bold";
-$opensans-extra-bold: "opensans-extra-bold";
-body {
-    font-family: $opensans-regular ;
-}
\ No newline at end of file
+/* // $opensans-regular: "opensans-regular";
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/assets/css/typography.css:64]: ignore , will use it in future

**Review by shalinee-walkover — COMMENTED** [review]
Approve

**Review by shalinee-walkover — APPROVED** [review]
Approve

**Review by rohit-walkover — APPROVED** [review]

**Review by aditi-tiwari17 — APPROVED** [review]

**Review by prateek-giddh — APPROVED** [review]

**Pull request merged**